packages/compass-sidebar/src/components/multiple-connections/icons/server-icon.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { css, palette, useDarkMode } from '@mongodb-js/compass-components';
+
+const iconStyles = css({
+  flex: 'none',
+});
+
+const ServerIcon = () => {
```

Comment:
This is a copy of the icon in saved-connections. We should be able to re-use more of the icons between the two, but I didn't want to create conflicts between the PRs, so I'd rather address this separately.